### PR TITLE
[XAI] Use explainable_ai_sdk for tf2 image notebooks.

### PR DESCRIPTION
**Pull request by frederick0329**
Leverage [explainable_ai_sdk](https://github.com/GoogleCloudPlatform/explainable_ai_sdk) to improve usability.
The following is improved.

- Metadata creation.

- Making explanation request.

- Visualize response.

### CODE REVIEW

**Comment by sirtorry** [comment]
feel free to skip "notebook run" to meet launch deadline

File: notebooks/samples/explanations/tf2/ai-explanations-image.ipynb [hunk 1]
```diff
@@ -1,1771 +1,1508 @@
 {
```
Comment (thedriftofwords) [inline notebooks/samples/explanations/tf2/ai-explanations-image.ipynb:1]: <p>Reset section title to "Generate explanation metadata"</p><p> </p><p>Change the "We can use..." sentence to:</p><p> </p><p>"You can use the [Explainable AI SDK](https://pypi.org/project/explainable-ai-sdk/1.0.5/)</p><p> </p><p>Can you also add a line to print the metadata that's created, before exporting it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/tf2/ai-explanations-image.ipynb:1]: <p>Change third sentence:</p><p> </p><p>"You can use the Explainable AI SDK to send explanation requests to your deployed model and visualize the explanations."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Comment (frederick0329) [reply notebooks/samples/explanations/tf2/ai-explanations-image.ipynb:1]: The metadata contains the baseline so it's huge.

File: notebooks/samples/explanations/tf2/ai-explanations-image.ipynb [hunk 2]
```diff
@@ -1,1771 +1,1636 @@
...
```
Comment (thedriftofwords) [inline notebooks/samples/explanations/tf2/ai-explanations-image.ipynb:1]: <p>You can use the <a href="https://pypi.org/project/explainable-ai-sdk/" target="_blank">Explainable AI SDK</a> to generate most of the fields <s>for us. </s></p><p> </p><p>Can you add a line of code that prints out the metadata before exporting it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (frederick0329) [reply notebooks/samples/explanations/tf2/ai-explanations-image.ipynb:1]: Done.

**Comment by thedriftofwords** [comment]
The changes work locally for Colab, but would still need some further debugging to pass the "notebook run" test. Merging now, because that isn't launch-blocking.

**Pull request merged**